packages/react-native-win32/src/Libraries/Image/Tests/SvgWin32Test.tsx

```diff
@@ -0,0 +1,97 @@
+import * as React from 'react';
```

Comment:
We don't want to be adding new Typescript files to this repo. I can link you to some discussion, but TLDR we should use flow here (FYI @FalseLobster )

Reply:
Based on some other files, it seems like this tester page should just be in a different repo in general since we shouldn't add a community module dependency to this one.

Reply:
Are we trying to move to flow in general here? I was kind of following the pattern with the ImageWin32Test.tsx which I checked in a few months ago. Is there a pattern somewhere else that we should follow for testing pages, and do we have any ideas for what to do with the existing test pages that are typescript?